ontopy/ontology.py

```diff
@@ -1468,18 +1468,27 @@ def closest_common_ancestor(*classes):
             "A closest common ancestor should always exist !"
         )
 
-    def get_ancestors(self, classes, include="all", strict=True):
+    def get_ancestors(
+        self,
+        classes: "Union[List, ThingClass]",
+        closest: bool = False,
+        generations: int = None,
+        strict: bool = True,
+    ) -> set:
         """Return ancestors of all classes in `classes`.
-        classes to be provided as list
-
-        The values of `include` may be:
-          - None: ignore this argument
-          - "all": Include all ancestors.
-          - "closest": Include all ancestors up to the closest common
-            ancestor of all classes.
-          - int: Include this number of ancestor levels.  Here `include`
-            may be an integer or a string that can be converted to int.
+        Args:
```

Comment:
```suggestion
        Arguments:
```

Reply:
Why: I have kept the style of the file. It says Args in all, and not Arguments.

Reply:
Then we keep Args.